### PR TITLE
Currency: Added build scripts for Nifi 1.22.0 for UBI 8 and Ubuntu 22.04

### PR DESCRIPTION
**Pull request by vishakadesai**
## Checklist

- [x] Have you checked and followed all the points mention in the [CONTRIBUTING.MD](https://github.com/ppc64le/build-scripts/blob/master/CONTRIBUTING.md)
- [x] Did you run the script(s) on fresh container with `set -e` option enabled and observe success ?
- [x] Did you have **Legal approvals** for patch files ?

### CODE REVIEW

**Comment by vishakadesai** [comment]
The Travis build is failing since it exceeds the log limit
Please find attached the build logs for Nifi 1.22.0
[nifi1.22.0-ubi8-output.txt](https://github.com/ppc64le/build-scripts/files/12023961/nifi1.22.0-ubi8-output.txt)

File: a/apache-nifi/build_info.json [hunk 1]
```diff
@@ -20,8 +20,13 @@
         "build_script" : "apache-nifi_1.17.0_ubi_8.5.sh"
     },
 
-    "rel/nifi-*.*.*": {
+    "rel/nifi-1.22.0": {
         "dir":"1.17.0_ubi_8",
```
Comment (nikhil-kalbande) [inline a/apache-nifi/build_info.json:24]: @vishakadesai Is this correct folder name?

File: a/apache-nifi/apache-nifi_1.22.0_ubi_8.5.sh [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+sed -i '/<artifactId>snappy-java<\/artifactId>/!b;n;c\\t\t<version>1.1.8</version>' pom.xml
+find="<additionalJOption>\-J\-Xmx512m<\/additionalJOption>"
+replace="<additionalJOptions>\
+<additionalJOption>\-J\-Xmx3g</additionalJOption>\
+<additionalJOption>\-J\-XX:+UseG1GC<\/additionalJOption>\
+<additionalJOption>\-J\-XX:ReservedCodeCacheSize=1g<\/additionalJOption>\
+<\/additionalJOptions>"
+sed -i "s#$find#$replace#g" pom.xml
+
+mvn install -Dmaven.test.skip=true
```
Comment (nikhil-kalbande) [inline a/apache-nifi/apache-nifi_1.22.0_ubi_8.5.sh:53]: Please add code to trap failure with exit code

File: a/apache-nifi/build_info.json (hunk 1 above)
Comment (vishakadesai) [reply a/apache-nifi/build_info.json]: made the corrections @nikhil-kalbande

File: a/apache-nifi/apache-nifi_1.22.0_ubi_8.5.sh (hunk 2 above)
Comment (vishakadesai) [reply a/apache-nifi/apache-nifi_1.22.0_ubi_8.5.sh]: Made the necessary additions @nikhil-kalbande

**Review by nikhil-kalbande — APPROVED** [review]

**Pull request merged**